components/shortestedge_collapse/wmtk/components/shortestedge_collapse/shortestedge_collapse.cpp

```diff
@@ -81,9 +77,9 @@ std::shared_ptr<Mesh> shortestedge_collapse(
 
     //////////////////////////////////
     // computing bbox diagonal
-    Eigen::VectorXd bmin(mesh.top_cell_dimension());
+    Eigen::VectorXd bmin(position_handle.dimension());
```

Comment:
we should have been using Eigen::AlignedBox for bounding boxes to begin with. There's at least 3 sorts of bugs that can happen with manual bbox initialization could have been avoided